indico/modules/events/abstracts/forms.py

```diff
@@ -171,6 +171,11 @@ class AbstractReviewingSettingsForm(IndicoForm):
                                                   widget=SwitchWidget(),
                                                   description=_('Enabling this allows submitters, authors, and '
                                                                 'speakers to also participate in the comments.'))
+    notify_on_new_comments = BooleanField(_('Comment notifications'),
+                                          [HiddenUnless('allow_comments', preserve_data=True)],
+                                          widget=SwitchWidget(),
+                                          description=_('Send an email notification to judges, conveners and reviewers '
+                                                        'when a new comment is added to an abstract.'))
```

Comment:
I'd go for something more concise like "Send email notifications when new comments are posted"